commit 3417ccbec7865821a662e34ca15b2683e739294e
Author: prudhvideep <prudhvideep1996@gmail.com>
Date:   Tue Jul 15 18:47:48 2025 -0400

    add test file to check ice candidates

diff --git a/src/App.tsx b/src/App.tsx
index e1c88d9..5dd547b 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -7,7 +7,6 @@ import {
 } from "@xyflow/react";
 
 import "@xyflow/react/dist/style.css";
-import { Message } from "./types/types";
 import { useState } from "react";
 import PeerNode from "./nodes/PeerNode";
 import { faker } from "@faker-js/faker";
@@ -15,10 +14,6 @@ import { FaBell, FaPaperPlane, FaVideo } from "react-icons/fa";
 import { IoCall, IoChatbubble } from "react-icons/io5";
 
 let peerId: string = "";
-let incomingFileData: any = null;
-const peerMap = new Map<string, RTCPeerConnection>();
-const channelMap = new Map<string, RTCDataChannel>();
-
 const nodeTypes = { peerNode: PeerNode };
 
 const generatePeerName = () => faker.animal.type() + "-" + faker.color.human();
@@ -34,8 +29,8 @@ if (!sessionStorage.getItem("peerId")) {
 }
 
 function App() {
-  const [nodes, setNodes, onNodesChange] = useNodesState<Node>([]);
-  const [messages, setMessages] = useState<string[]>([]);
+  const [nodes, _, onNodesChange] = useNodesState<Node>([]);
+  const [messages] = useState<string[]>([]);
   const [inputMessage, setInputMessage] = useState("");
 
 
diff --git a/src/main.tsx b/src/main.tsx
index 5693a08..75bd6c0 100644
--- a/src/main.tsx
+++ b/src/main.tsx
@@ -1,7 +1,6 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
 import './index.css'
-import App from './App.tsx'
 import Test from './Test.tsx'
 
 createRoot(document.getElementById('root')!).render(
